modules/op-adding-resources-in-database-for-tekton-hub.adoc

```diff
@@ -0,0 +1,48 @@
+// This module is included in the following assembly:
+//
+// *cicd/pipelines/using-tekton-hub-with-openshift-pipelines.adoc
+
+:_content-type: PROCEDURE
+[id="adding-resources-in-the-database-for-tekton-hub_{context}"]
+= Adding resources in the database for {tekton-hub}
```

Comment:
What does "adding resources" mean to the admin?

Reply:
Adding resources means adding all tasks and pipelines in the database present in the catalog which is reflected on the Hub

Reply:
We should make the title more reflective of that IMO